charts/zookeeper/README.md

```diff
@@ -66,7 +66,7 @@ The following table lists the configurable parameters of the Zookeeper chart and
 | `pod.tolerations` | Specifies the pod's tolerations | `[]` |
 | `pod.env` | List of environment variables to set in the container | `[]` |
 | `pod.annotations` | Specifies the annotations to attach to pods | `{}` |
-| `pod.securityContext` | Specifies the security context for the entire pod | |
+| `pod.securityContext` | Specifies the security context for the entire pod | `{}` |
 | `pod.terminationGracePeriodSeconds` | Amount of time given to the pod to shutdown normally | `180` |
```

Comment:
Please change termination grace period default value to 30 seconds

Reply:
Done